skyportal/handlers/api/public_pages/public_source_page.py

```diff
@@ -51,6 +58,44 @@ def get_redshift_to_display(source):
     return redshift_display
 
 
+def get_photometry(data_to_publish, source_id, group_ids, stream_ids, session):
+    stmt = Photometry.select(session.user_or_token, mode="read").where(
+        Photometry.obj_id == source_id
+    )
+    if len(group_ids) and len(stream_ids):
```

Comment:
Why just `if len(...)`? Is it just because you are checking if this lists aren't empty? If so I would put the `== 0` to keep it as explicit as possible.